packages/form/src/Field.js

```diff
@@ -57,6 +57,7 @@ const Field = ({
       readOnly={readOnly}
       feedback
       help={!!helpMessage}
+      aria-describedby={`${id}-error`}
```

Comment:
We should make this one conditional as well. You can import `useField` from `formik` to get the error state.

```
const [, { error }] = useField(id);
```

Then we can do use that error to check if we should pass in the id.